nautobot/project-static/css/base.css

```diff
@@ -320,6 +320,9 @@ table.report th a {
     border-bottom-left-radius: 9px;
     border-bottom-right-radius: 9px;
 }
+.panel {
+    overflow: hidden  /* Fix for panel overflow issues */
+}
```

Comment:
This seems to cause integration test to fail, is there a way to fix it another way?
Maybe?
```suggestion
.panel {
    overflow: scroll  /* Fix for panel overflow issues */
}
```

Reply:
That breaks all of the panels for me
![image](https://github.com/user-attachments/assets/b94652d1-ac5b-46f0-9874-5bd545c69e12)

Reply:
Maybe we should make that panel wider?

![image](https://github.com/user-attachments/assets/46c835bd-2c89-412e-937e-d21eedce6636)

Reply:
This is what it looks like without the css change I made

![image](https://github.com/user-attachments/assets/769fbae3-ee4e-4ce4-b51f-5d88fcf1c54d)

Reply:
The `test_modal_rendered` test keeps failing for me locally even with these suggestions.

Reply:
Yeah I like the idea of making the panel wider

Reply:
Here's another example of a currently broken panel. This is in the cluster "remove devices" view

Before my css change

![image](https://github.com/user-attachments/assets/7bf4f99e-120c-4aea-b8f3-605492627dc9)

After

![image](https://github.com/user-attachments/assets/90bf2df5-67f1-4ec6-a66d-61d728176a8b)


Then if you make the window a bit wider

![image](https://github.com/user-attachments/assets/92261041-b24e-4007-8590-8aab2f8d73ba)

This is probably another good candidate for the #5313 fixes

Reply:
Tracking the global fix with #5948